File: app/src/main/java/info/xudshen/android/playground/model/SampleItemModel.java
package info.xudshen.android.playground.model;

/**
 * Created by xudong on 2017/1/11.
 */

public class SampleItemModel {
    private String title;
    private Class clazz;

    public SampleItemModel(String title, Class clazz) {
        this.title = title;
        this.clazz = clazz;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public Class getClazz() {
        return clazz;
    }

    public void setClazz(Class clazz) {
        this.clazz = clazz;
    }
}

File: app/src/main/java/info/xudshen/android/playground/model/SampleItemModel.kt
package info.xudshen.android.playground.model

import android.support.annotation.DrawableRes
import android.support.v4.app.Fragment

/**
 * Created by xudong on 2017/1/11.
 */

class SampleItemModel(val title: String, @DrawableRes val cover: Int = 0, val clazz: Class<out Fragment>)
